community/consistency-check/src/main/java/org/neo4j/consistency/store/synthetic/CountsEntry.java

```diff
@@ -30,14 +30,36 @@
  */
 public class CountsEntry extends AbstractBaseRecord
 {
-    private final CountsKey key;
-    private final long count;
+    private CountsKey key;
+    private long count;
+
+    // ========= TRANSITIONAL CONSTRUCTORS =============
+    public CountsEntry()
+    {
+        super( -1 );
+    }
 
     public CountsEntry( CountsKey key, long count )
     {
+        super( -1 );
+        initialize( key, count );
+    }
+    // ========= TRANSITIONAL CONSTRUCTORS =============
+
+    public CountsEntry initialize( CountsKey key, long count )
+    {
+        super.initialize( true );
```

Comment:
since we can use clear() we do not need `initialize( true )` it can be simplified to `initialize()`